src/assets.rs

```diff
@@ -284,6 +284,10 @@ impl AssetLoader for FighterLoader {
                     .with_dependency(texture_path),
                 );
                 meta.spritesheet.atlas_handle.push(atlas_handle);
+                meta.size = Vec2::new(
+                    meta.spritesheet.tile_size.x as f32 / 2.,
+                    meta.spritesheet.tile_size.y as f32 / 2.,
+                );
```

Comment:
In order for this to work, we need to enforce the sheets to have 0 or close to padding, which is fine just something to remember.

Reply:
I noticed that all of the sorting seems correct except for the boss. With this it seems that its possible to walk slightly up from the boss' feet and still be in front, do you think its due to padding in that case?

Reply:
Oh now that I'm looking again I may have misinterpreted how size was being used/determined before. I thought it was = to half the sprite size, but I see that wasn't the case for every fighter. How did you determine the values for size previously?

Reply:
Manually checking the actual size of the sprite.

Reply:
Alright Ill revert those changes for now but we should probably switch to enforcing the same padding for every sprite in future.

Reply:
Probably just a mistake on my part but the x value is not really used right now.

Reply:
Ok were you doing anything special to account for padding?

Reply:
Nothing special just checked in-game if the debug line would be at their feet. In the case of the brute skins sometimes it would not fit right because of https://github.com/fishfolks/punchy/pull/230#issuecomment-1214220775.
It might be a good idea to enforce the same padding. We could go as far as to create a simple tool that makes the sprite sheet and the YAML file.